cloudformation/cloudformation.yaml

```diff
@@ -8,7 +8,84 @@ Parameters:
             - PROD
             - CODE
         Default: CODE
+Mappings:
+  EnvironmentBucketMap:
+    CODE:
+      "logs": "fulfilment-s3-logs-code"
+      "export": "fulfilment-export-code"
+    PROD:
+      "logs": "fulfilment-s3-logs-prod"
+      "export": "fulfilment-export-prod"
 Resources:
+  EncryptBucketPolicy:
+    Properties:
+      Bucket: FulfilmentBucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Sid: DenyIncorrectEncryptionHeader
+          Effect: Deny
+          Principal: "*"
+          Action: s3:PutObject
+          Resource: arn:aws:s3:::YourBucket/*
```

Comment:
this resource name doesn't look right, is it just a placeholder

Reply:
![](https://media.giphy.com/media/3ov9jCGzR1JtD8Hlni/giphy.gif) oops